package.json

```diff
@@ -32,6 +32,27 @@
     "ext": "js",
     "exec": "babel-node ./src/server/"
   },
+  "eslintConfig": {
+    "extends": [
+      "eslint:recommended",
+      "plugin:react/recommended"
+    ],
+    "rules": {    
+      "react/react-in-jsx-scope": "off",
```

Comment:
Also found here https://medium.com/@joelmasters/setting-up-eslint-for-nextjs-37163d4cabaa

Reply:
I'm not sure I agree with this setting (or the `"globals": { "React": "writable"}` setting).

If we are only ever going to use Next.js, then this looks convenient, but if we ever want to switch to something else, or if we want to port our React components somewhere else, I think this could this break compatibility.

Reply:
Next.js doesn't require us to import React in pages (which we don't have many of yet), and ESLint will start barking about that unless we turn off the react-in-jsx-scope setting. If we migrate away from Next.js then we can remove it, but I think we're going to have annoyances otherwise.

I'll try removing the globals stuff and see what happens.

Reply:
This has been removed now.